Ssmpu/Memory_Protection_Unit.adoc

```diff
@@ -37,9 +37,12 @@ image::RV32_MPU_address_register_format.png[title="MPU address register format,
 image::RV64_MPU_address_register_format.png[title="MPU address register format, RV64"]
 
 The layout of MPU configuration registers is the same as PMP configuration registers, as is shown in Figure 6.
-The X/W/R field controls instruction execution, write and read permissions.
+The X/W/R bitsd controls instruction execution, write and read permissions.
```

Comment:
bitsd -> bits

Reply:
Thanks for pointing it out, will fix it.